docs/monitoring-usage.rst

```diff
@@ -24,8 +24,13 @@ Essential concepts:
 Please refer to the documentation for the `Monitoring API`_ for
 more information.
 
-At present, this client library supports querying of time series,
-metric descriptors, and resource descriptors.
+At present, this client library supports the following features
+of the API:
+
+- Querying of time series.
+- Querying of metric descriptors and resource descriptors.
```

Comment:
*monitored* resource descriptors.
(To match documentation on cloud.google.com.)